skops/io/tests/_utils.py

```diff
@@ -173,3 +178,62 @@ def assert_method_outputs_equal(estimator, loaded, X):
             X_out1 = getattr(estimator, method)(X)
             X_out2 = getattr(loaded, method)(X)
             assert_allclose_dense_sparse(X_out1, X_out2, err_msg=err_msg, atol=ATOL)
+
+
+def downgrade_state(*, data: bytes, keys: list[str], old_state: dict, protocol: int):
```

Comment:
this is a bit cryptic, an example in the docstring on how it's supposed to be used would be nice.